dev/io.openliberty.microprofile.config.internal.common/resources/io/openliberty/microprofile/config/internal/common/resources/MPConfigCommonMessages.nlsprops

```diff
@@ -1,28 +1,29 @@
 # are made available under the terms of the Eclipse Public License 2.0
 # which accompanies this distribution, and is available at
 # http://www.eclipse.org/legal/epl-2.0/
 # 
 # SPDX-License-Identifier: EPL-2.0
-#
-# Contributors:
-#     IBM Corporation - initial API and implementation
 ###############################################################################
-#CMVCPATHNAME io.openliberty.io.smallrye.config/resources/io/openliberty/io/smallrye/config/resources/SmallryeConfigMessages.nlsprops
+#CMVCPATHNAME io.openliberty.microprofile.config.internal.common/resources/io/openliberty/microprofile/config/internal/common/resources/MPConfigCommonMessages.nlsprops
 #ISMESSAGEFILE TRUE
 #NLS_ENCODING=UNICODE
 
 #COMPONENTPREFIX CWWKC                 
-#COMPONENTNAMEFOR CWWKC Smallrye Config
-
+#COMPONENTNAMEFOR CWWKC MicroProfile Config with InstantOn
 # Message numbers CWWKC0651-700 are reserved for this file.
+
+#COMPONENTPREFIX CWMCG                 
+#COMPONENTNAMEFOR CWMCG MicroProfile Config Common
+# Message numbers CWMCG0400-499 are reserved for this file.
+
 # NLS_MESSAGEFORMAT_VAR                
-# MSG_DESCRIPTIVE_NAME_CWWKC0000=CWWKC0000I: This is a message with inserts {0}
-# MSG_DESCRIPTIVE_NAME_CWWKC0000.explanation=Explanation text for the message
-# MSG_DESCRIPTIVE_NAME_CWWKC0000.useraction=User action text for the message
+# MSG_DESCRIPTIVE_NAME_CWMCG0000=CWMCG0000I: This is a message with inserts {0}
+# MSG_DESCRIPTIVE_NAME_CWMCG0000.explanation=Explanation text for the message
+# MSG_DESCRIPTIVE_NAME_CWMCG0000.useraction=User action text for the message
 
-WARNING_UPDATED_CONFIG_PROPERTY_NOT_USED_CWWKC0651=CWWKC0651W: The MicroProfile configuration value for the {0} key has changed since the checkpoint action completed on the server. If the value of the key changes after the checkpoint action, the application might not use the updated value.
-WARNING_UPDATED_CONFIG_PROPERTY_NOT_USED_CWWKC0651.explanation=The configuration value was accessed during a checkpoint action and updated before the server process got restored. Depending on how the application accesses the configuration, the updated value might not be used.
-WARNING_UPDATED_CONFIG_PROPERTY_NOT_USED_CWWKC0651.useraction=Avoid accessing configuration properties during checkpoint.
+WARNING_UPDATED_CONFIG_PROPERTY_NOT_USED_CWMCG0651=CWWKC0651W: The MicroProfile configuration value for the {0} key has changed since the checkpoint action completed on the server. If the value of the key changes after the checkpoint action, the application might not use the updated value.
+WARNING_UPDATED_CONFIG_PROPERTY_NOT_USED_CWMCG0651.explanation=The configuration value was accessed during a checkpoint action and updated before the server process got restored. Depending on how the application accesses the configuration, the updated value might not be used.
+WARNING_UPDATED_CONFIG_PROPERTY_NOT_USED_CWMCG0651.useraction=Avoid accessing configuration properties during checkpoint.
```

Comment:
I don't think you want to change the message code here.
- This is a Config + InstantOn message
- There's no corresponding update to the code or to the other language NLS files

Reply:
ack! no, that is a mistake!